quantized_mesh_tile/editable_terrain.py

```diff
@@ -0,0 +1,487 @@
+
+        if len(indices) == 0:
+            raise Exception("No edge vertices found for edge: {}".format(edge))
+        return indices
+
+    def get_edge_coordinates(self, edge):
+        # type: (string) -> Array
+        """
+        Returns a array of coordinate-tupels for all vertices on the specified edge
+        :return: Array of 3d coordinates (3-tupel of float; x,y,z)
+        :param edge: the edge of the tile. Specified by the flags 'w','n','e','s'
+                    for west-, north-, east- and  south-edge
+        """
+        edge_coordinates = []
+        coordinates = self.getVerticesCoordinates()
+        vertices = self.get_edge_vertices(edge)
+        for vertex in vertices:
+            edge_coordinates.append(coordinates[vertex])
+
+        return edge_coordinates
+
+    @property
+    def get_bounding_box(self):
+        """
+        Returns the bounding box of the tile in WGS84 degree
+        :return: Dictionary of floats for boundingbox of the tile with
+                keys ['west','east','north', 'south']
+        """
+        return {'west': self._west,
+                'east': self._east,
+                'north': self._north,
+                'south': self._south}
+
+    def set_normal(self, index, normal):
+        # type: (int, Array) -> void
+        """
+        Sets the normal vector for the vertex which is specified with the index,
+        changing a normal vector causes a rebuild of all indices
+        :param index: index of the vertex
+        :param normal: the normal vector in the form [x,y,z]
+        """
+        self.is_index_dirty = True
+        self.vLight[index] = normal
+
+    def set_height(self, index, height):
+        # type: (int, float) -> void
+        """
+        Sets the height for the vertex  which is specified with the index,
+        if the new height is out of the tile height range (min/max height),
+        all heights are requantized during save-process
+        :param index: index of the vertex
+        :param height:the height at the specified vertex index
+        """
+        height_is_dirty = False
+
+        if height < self.header['minimumHeight']:
+            height_is_dirty = True
+        if self.header['maximumHeight'] < height:
+            height_is_dirty = True
+
+        if height_is_dirty or self._changed_heights:
+            if not self._changed_heights:
+                self._changed_heights = [self._dequantize_height(x) for x in self.h]
+            self._changed_heights[index] = height
+        else:
+            self.h[index] = self._quantize_height(height)
+
+    def get_height(self, index):
+        # type: (int) -> float
+        """
+        Returns the dequantized height at the specified vertex index
+        :param index: index of the vertex
+        :return: the height at the specified vertex
+        """
+        height = self._dequantize_height(self.h[index])
+        return height
```

Comment:
`getHeightAt` ?

```python
return self._dequantizeHeightAt(self.height[index])
```

Reply:
the passed argument for self._dequantizeHeight(self.height[index]) is a height-value and not a index-value, the idea for this method was, to isolate/seperate the more complex dequantize-operation from the index-lookup stuff...

..but i can live with self._dequantizeHeightAt(index)

Reply:
..uh..got it :)